src/_pytest/terminal.py

```diff
@@ -628,6 +628,24 @@ def pytest_runtest_logreport(self, report: TestReport) -> None:
             else:
                 markup = {}
         self._progress_nodeids_reported.add(rep.nodeid)
+
+        # Calculate and send progress information to terminal tab
+        if self._show_progress_info:
+            total_items = 0
+            if hasattr(self, "_session") and self._session is not None:
+                if hasattr(self._session, "items"):
+                    total_items = len(self._session.items)
```

Comment:
I don't think we need the `hasattr` checks given `_session` is set during `__init__` (same for `Session.items`:



```suggestion
            total_items = len(self._session.items) if self._session is not None else 0
```